### ISSUE TITLE
Missing modules

### ISSUE THREAD

**Issue by SiftingSands** [skyhehe123/SA-SSD#9]
I followed the installation instructions, but had to make the following changes when running `python3 tools/create_data.py`.

*Install pybind11
*Install numba
*Instal mmcv
*Install terminaltables
*Moved "create_data.py" up one directory into ~/SA-SSD/ instead of ~/SA-SSD/tools. Otherwise, it wouldn't load modules from mmdet.

I'm not sure if this is intended, or if I messed up installation somewhere.

**Comment by SiftingSands** [comment]
Also getting the following issue with imports.

```
  File "<REDACTED>/SA-SSD/mmdet/core/evaluation/eval_hooks.py", line 17, in <module>
    import tools.kitti_common as kitti
ModuleNotFoundError: No module named 'tools.kitti_common'
```

Again, maybe I screwed up the install here?

**Comment by zwqnju** [comment]
Maybe you should not move create_data.py. 
Just run `python tools/create_data.py ........` in the root directory.

**Comment by SiftingSands** [comment]
> Maybe you should not move create_data.py.
> Just run `python tools/create_data.py ........` in the root directory.

That's what I tried at first, otherwise I wouldn't have moved it if it worked...
It fails to import "mmdet" for example in the default location.  Should "mmdet" show up in "pip list"? I did "python3 setup.py build_ext --inplace" for every folder in "mmdet/ops" per the instructions.

**Comment by SiftingSands** [comment]
Closing this issue. My build somehow started working after retrying in a few days...

Also, I ran into the following error :

> SA-SSD/mmdet/models/single_stage_heads/ssd_rotate_head.py", line 322, in get_guided_anchors
    opp_labels = (box_preds[..., -1] > 0) ^ dir_labels.byte()
RuntimeError: result type Byte can't be cast to the desired output type Bool

I'm using PyTorch 1.4, so it's probably a compatibility issue with different versions. Made the following changes to also avoid the following warning.

> /pytorch/aten/src/ATen/native/IndexingUtils.h:20: UserWarning: indexing with dtype torch.uint8 is now deprecated, please use a dtype torch.bool instead.

`opp_labels = (box_preds[..., -1] > 0).byte() ^ dir_labels.byte() 
  box_preds[opp_labels.bool(), -1] += np.pi`
